src/components/TopBar/CallButton.vue

```diff
@@ -352,6 +357,12 @@ export default {
 		},
 	},
 
+	watch: {
+		token() {
+			this.$store.dispatch('resetCallHasJustEnded')
+		}
```

Comment:
User can just go to another conversation and come back to start a call :p 
Breaking the flag by token in store should solve this

Reply:
I don't think that's a problem, as a goal is to prevent accidental click. If user switch back and forth, he probably knows what he's doing

Reply:
I wouldn't really care about that. I think the biggest usecase of the whole thing is that people don't accidentally restart a call by clicking the "Start call" button which just now replaced "Leave call", when a moderator already made them leave by using "End call for everyone".
So changing the room back and forth is totally fine to call again.